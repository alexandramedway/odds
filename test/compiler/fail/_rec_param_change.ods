do f = (x) ->
    do f(2)
    return !f(true)
