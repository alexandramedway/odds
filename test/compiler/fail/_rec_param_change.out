Analyzer Exception: Function 'f' expected argument of type Num but was passed Bool instead
